src/Widgets/ButtonWidget/ButtonWidgetEditingConfig.js

```diff
@@ -32,4 +32,27 @@ Scrivito.provideEditingConfig("ButtonWidget", {
     alignment: "left",
     style: "btn-primary",
   },
+  validations: [
+    [
+      "target",
+
+      target => {
+        if (!target) {
+          return {
+            message: "The link target should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+    [
+      "title",
+
+      title => {
+        if (!title) {
+          return { message: "The title is empty.", severity: "info" };
```

Comment:
Let's make the message a bit more actionable:

```suggestion
          return {
            message: "Providing the button title is recommended.",
            severity: "info",
          };
```